jetty-core/jetty-client/src/main/java/org/eclipse/jetty/client/transport/internal/HttpReceiverOverHTTP.java

```diff
@@ -126,6 +127,7 @@ public Content.Chunk read(boolean fillInterestIfNeeded)
         if (LOG.isDebugEnabled())
             LOG.debug("ParseAndFill needFillInterest {} in {}", needFillInterest, this);
         chunk = consumeChunk();
+        runCompletionTaskIfAny();
```

Comment:
This aims to be generic, but it does not need to.
I think it's clearer to call `responseSuccess()` directly from here.

Reply:
Done.